tests/integration/test_gardenlinux.py

```diff
@@ -378,7 +394,7 @@ def test_aws_ena_driver(client, aws):
     assert exit_code == 0, f"no {error=} expected"
     assert output.rstrip() == "ena", "Expected network interface to run with ena driver"
 
-def test_apparmor(client):
+def test_apparmor(client, non_chroot, non_kvm):
```

Comment:
Please rebase to latest master as the change introduced in PR#766 should make this test succeed even on KVM.

Reply:
I tested the kvm test with the kvm image, the kvm image is build without the gardener, chost and khost feature. The gardener, chost and khost feature install apparmor .
I tested with the gcp-gardener image with the kvm image and the apparmor test passes. Since the test is connected to features that are not enabled in all images we need a test to check if a feature that installs apparmor is enabled.
Or maybe we can check in the apparmor test first if apparmor is installed, if not the test passes, too.

Reply:
Ok, I believe that the test fails because `aa-enabled` is not installed in the image. Maybe we should change the test to work like the SELinux test by looking into `/sys/kernel/security/lsm` instead. If the image runs with SELinux, this test should not fail because of a missing `aa-enabled` but actually succeed.